packages/verification/tests/rest/v1/callbacks/webhooks-events.steps.ts

```diff
@@ -0,0 +1,82 @@
+import { VerificationCallbackWebhooks, Verification } from '../../../../src';
+import { Given, Then, When } from '@cucumber/cucumber';
+import assert from 'assert';
+import { IncomingHttpHeaders } from 'http';
+
+let verificationCallbackWebhook: VerificationCallbackWebhooks;
+let rawEvent: any;
+let event: Verification.VerificationCallbackEvent;
+let formattedHeaders: IncomingHttpHeaders;
+
+const processEvent = async (response: Response) => {
+  formattedHeaders = {};
+  response.headers.forEach((value, name) => {
+    formattedHeaders[name.toLowerCase()] = value;
+  });
+  rawEvent = await response.text();
+  rawEvent = rawEvent.replace(/\s+/g, '');
```

Comment:
`rawEvent` is no more raw 🤔 
Is authentication is failingif using the true 'raw' coming from `response.text()`?

Reply:
Yes, this is a compromise I did to keep a nice readability for the mockserver JSON files: the body comes with the formatting (spaces and carriage returns), which is not what the server will actually return. This function ensures the data is the same as the one we will get on a webhook before it's being processed.

Reply:
No way to have mockerserver sources containing readable JSON payload but stripped from spaces before sending them back?
So we could have both:
- nice JSON
- server-like responses sent back

Reply:
After some tries, I noticed that despite the response is minified in the mockserver config, the mockserver will send pretty print it before sending it on the network.
As on the SDK side, we apply strictly the algorithm, I'll update the signatures on the mockserver side and this line won't be necessary anymore: in the end, the data will be prettified all over the places: WYSIWYG

Reply:
Removed with the commit [b4f61d7](https://github.com/sinch/sinch-sdk-node/pull/131/commits/b4f61d7e1091bde10c5c0975abfc770953db9494#diff-454750f6a252b89d1c7ef0eae82fa7d76faaaa197ade4a8ef417a3dbc029cac1L17)